frontend/test/pytest/test_gradient.py

```diff
@@ -688,5 +688,44 @@ def interpret_grad(params):
     assert np.allclose(compile_grad(jnp.array(inp)), interpret_grad(inp))
 
 
+@pytest.mark.parametrize("diff_method", ["fd", "adj", "ps"])
+@pytest.mark.parametrize("inp", [(1.0), (2.0), (3.0), (4.0)])
+def test_finite_diff_multiple_devices(inp, diff_method, backend):
```

Comment:
@dime10 what do you think about this test and the other one in `test_qnode.py`?

Reply:
Nice 👍 Fixing one device to lightning.qubit and keeping the other variable also solves the issue that if two devices were hardcoded, the test couldn't run if kokkos was not installed.
An alternative might have been to use the pytest skip feature upon detecting that kokkos is not available.